src/main/java/usersandpostsCRUD/demo/service/UserService.java

```diff
@@ -0,0 +1,87 @@
+package usersandpostsCRUD.demo.service;
+
+import org.springframework.stereotype.Service;
+import usersandpostsCRUD.demo.dto.UserDto;
+import usersandpostsCRUD.demo.dto.UserResponseBody;
+import usersandpostsCRUD.demo.entity.User;
+import usersandpostsCRUD.demo.exception.UserNotFoundException;
+import usersandpostsCRUD.demo.repository.UserRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public List<User> getAllUsers() {
```

Comment:
You return DTO from the service level. Not the dabatase table, adjust everywhere